src/Plugin/DirectoryModel/SetStateRequired.php

```diff
@@ -113,11 +113,13 @@ private function getStateRequiredCountries(): array
     {
         $store = $this->getStore();
         if (!$this->stateRequiredCountries) {
-            $this->stateRequiredCountries = explode(',', $this->scopeConfig->getValue(
+            $configStateRequired = $this->scopeConfig->getValue(
                 'general/region/state_required',
                 ScopeInterface::SCOPE_STORES,
                 $store->getCode()
-            ));
+            );
+            if (!$configStateRequired) return [];
```

Comment:
That will work but let's always save the value in `$this->stateRequiredCountries`

A more simple fix would be:
```
            $this->stateRequiredCountries = explode(',', (string) $this->scopeConfig->getValue(
                'general/region/state_required',
                ScopeInterface::SCOPE_STORES,
                $store->getCode()
            ));
```

no additional variables, still a single return point